uint8 role
uint8 id
float32 dis
float32 fpRssi
float32 rxRssi
